[2025-01-31T01:47:59.693+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T01:47:59.704+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: quarta_dag.tsk3 manual__2025-01-31T01:47:57.962187+00:00 [queued]>
[2025-01-31T01:47:59.708+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: quarta_dag.tsk3 manual__2025-01-31T01:47:57.962187+00:00 [queued]>
[2025-01-31T01:47:59.708+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-31T01:47:59.717+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): tsk3> on 2025-01-31 01:47:57.962187+00:00
[2025-01-31T01:47:59.721+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3263) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T01:47:59.722+0000] {standard_task_runner.py:72} INFO - Started process 3264 to run task
[2025-01-31T01:47:59.723+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'quarta_dag', 'tsk3', 'manual__2025-01-31T01:47:57.962187+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/quarta_dag.py', '--cfg-path', '/tmp/tmpc6l0pbgx']
[2025-01-31T01:47:59.724+0000] {standard_task_runner.py:105} INFO - Job 38: Subtask tsk3
[2025-01-31T01:47:59.769+0000] {task_command.py:467} INFO - Running <TaskInstance: quarta_dag.tsk3 manual__2025-01-31T01:47:57.962187+00:00 [running]> on host 15928652fac1
[2025-01-31T01:47:59.841+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='quarta_dag' AIRFLOW_CTX_TASK_ID='tsk3' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T01:47:57.962187+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-31T01:47:57.962187+00:00'
[2025-01-31T01:47:59.842+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T01:47:59.859+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-31T01:47:59.861+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-01-31T01:47:59.872+0000] {subprocess.py:99} INFO - Output:
[2025-01-31T01:48:04.879+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-31T01:48:04.946+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T01:48:04.947+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=quarta_dag, task_id=tsk3, run_id=manual__2025-01-31T01:47:57.962187+00:00, execution_date=20250131T014757, start_date=20250131T014759, end_date=20250131T014804
[2025-01-31T01:48:05.006+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T01:48:05.042+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-31T01:48:05.043+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
